packages/web-components/src/toolbar/toolbar.spec.md

```diff
@@ -0,0 +1,88 @@
+# Toolbar
+
+## Component Description
+
+A toolbar is a container for grouping a set of controls, such as buttons, menubuttons, or checkboxes.
+
+When a set of controls is visually presented as a group, the toolbar role can be used to communicate the presence and purpose of the grouping to screen reader users. Grouping controls into toolbars can also be an effective way of reducing the number of tab stops in the keyboard interface.
+
+## Design Spec
+
+[Link to Design Spec in Figma](https://www.figma.com/file/QfHyx6ApwFEcw5q4pM9CSZ/Toolbar?node-id=2351-6672&t=MuajdFxoCBjwlK4w-0)
+
+[Link to Fluent 2 Storybook](https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-toolbar--default)
+
+## Engineering Spec
+
+### Inputs
+
+- @attr orientation: "horizontal" "vertical" | "horizontal"
+- @attr size: "small" "medium" "large" | "medium"
+
+### Outputs
+
+### Slots
+
+- label: Content which appears as the label for the toolbar.
+- start: Content which appears at the start of the toolbar, before the default slotted items.
+- default: Content which appears in the default location of the toolbar, left-aligned.
+- end: Content which appears at the end of the toolbar, right aligned.
+
+### CSS Variables
+
+None
+
+## Interactions
+
+Interactions are inherited from FAST Toolbar component.
+
+| Name             | Description                             |
+| ---------------- | --------------------------------------- |
+| `clickHandler`   | Sets the active index                   |
+| `focusinHandler` | Sets focus on element with active index |
+| `keydownHandler` | Handles keydown events on the toolbar   |
+
+## Accessibility
+
+- [x] Find the matching component through WCAG's patterns: https://www.w3.org/WAI/ARIA/apg/patterns/
+  - https://www.w3.org/WAI/ARIA/apg/patterns/toolbar.html
+- [x] Are there any accessibility elements unique to this component?
+- [x] List ARIA attributes
+  - [x] role="toolbar" in FAST
+  - [x] aria-orientation="vertical" in FAST
+- [ ] Does the component support 400% zoom?
```

Comment:
While this box unchecked, it is worth noting that the items do overflow their container at 400% zoom:

<img width="1323" alt="image" src="https://github.com/microsoft/fluentui/assets/5067638/64f7aa09-8b31-433f-ae3d-c36fb5eb7b03">

However, this is also true at 100% zoom with a slightly narrower container:

<img width="715" alt="image" src="https://github.com/microsoft/fluentui/assets/5067638/73ebf141-cb30-489b-b624-70d0736447a8">

Reply:
I included this caveat in my initial description:
Items present in the [Fluent UI Figma (MVP)](https://www.figma.com/file/QfHyx6ApwFEcw5q4pM9CSZ/Toolbar?type=design&node-id=2367-13918&t=NuPLsJ1NemOOP6dw-4) but not implemented. These will be added in the next Sprint:

- Responsive toolbar which implements a More menu and ellipses
- Multiple toolbar groups server as tab stops within the toolbar